packages/contracts-bedrock/test/L1/StandardValidator.t.sol

```diff
@@ -106,911 +104,622 @@ contract StandardValidatorTest is CommonTest {
+    function test_validate_permissionedDisputeGameInvalidClockExtension_succeeds() public {
         vm.mockCall(
-            address(_disputeGame),
-            abi.encodeCall(IPermissionedDisputeGame.clockExtension, ()),
-            abi.encode(Duration.wrap(1000))
+            address(pdg), abi.encodeCall(IPermissionedDisputeGame.clockExtension, ()), abi.encode(Duration.wrap(1000))
         );
-        assertEq(string.concat(errorPrefix, "-80"), validate(true));
+        assertEq("PDDG-80", _validate(true));
+    }
 
-        // Test invalid splitDepth
-        _mockValidationCalls();
-        vm.mockCall(address(_disputeGame), abi.encodeCall(IPermissionedDisputeGame.splitDepth, ()), abi.encode(20));
-        assertEq(string.concat(errorPrefix, "-90"), validate(true));
+    /// @notice Tests that the validate function successfully returns the right error when the
+    ///         PermissionedDisputeGame splitDepth is invalid.
+    function test_validate_permissionedDisputeGameInvalidSplitDepth_succeeds() public {
+        vm.mockCall(address(pdg), abi.encodeCall(IPermissionedDisputeGame.splitDepth, ()), abi.encode(20));
+        assertEq("PDDG-90", _validate(true));
+    }
 
-        // Test invalid maxGameDepth
-        _mockValidationCalls();
-        vm.mockCall(address(_disputeGame), abi.encodeCall(IPermissionedDisputeGame.maxGameDepth, ()), abi.encode(50));
-        assertEq(string.concat(errorPrefix, "-100"), validate(true));
+    /// @notice Tests that the validate function successfully returns the right error when the
+    ///         PermissionedDisputeGame maxGameDepth is invalid.
+    function test_validate_permissionedDisputeGameInvalidMaxGameDepth_succeeds() public {
+        vm.mockCall(address(pdg), abi.encodeCall(IPermissionedDisputeGame.maxGameDepth, ()), abi.encode(50));
+        assertEq("PDDG-100", _validate(true));
+    }
 
-        // Test invalid maxClockDuration
-        _mockValidationCalls();
+    /// @notice Tests that the validate function successfully returns the right error when the
+    ///         PermissionedDisputeGame maxClockDuration is invalid.
+    function test_validate_permissionedDisputeGameInvalidMaxClockDuration_succeeds() public {
         vm.mockCall(
-            address(_disputeGame),
-            abi.encodeCall(IPermissionedDisputeGame.maxClockDuration, ()),
-            abi.encode(Duration.wrap(1000))
+            address(pdg), abi.encodeCall(IPermissionedDisputeGame.maxClockDuration, ()), abi.encode(Duration.wrap(1000))
         );
-        assertEq(string.concat(errorPrefix, "-110"), validate(true));
-
-        if (_gameType.raw() == GameTypes.PERMISSIONED_CANNON.raw()) {
-            _mockValidationCalls();
-            vm.mockCall(
-                address(_disputeGame),
-                abi.encodeCall(IPermissionedDisputeGame.challenger, ()),
-                abi.encode(address(0xbad))
-            );
-            assertEq(string.concat(errorPrefix, "-120"), validate(true));
```

Comment:
this check is implemented for the permissioned but not permissionless game.

Reply:
PDDG-120 is for the challenger which doesn't exist for the permissionless game